coalib/bearlib/aspects/Smell.py

```diff
@@ -256,16 +462,14 @@ class docs:
 @Smell.subaspect
 class Naming:
     """
-    This aspect checks on identifiers in your codebase (their length
-    and the appropriate naming convention to use for them, be it variables,
-    classes or functions names.)
+    `Naming` refers to the naming conventions to use for identifiers.
     """
     class docs:
         example = """
         camelCase naming convention, snake_case naming convention,
         hyphenated-case naming convention etc...
         """
-        example_language = 'English'
+        example_language = 'rst'
```

Comment:
This example isnt rst syntax, and rst isnt a particularly good language to show naming convention problems.